generated/nidcpower/README.rst

```diff
@@ -42,25 +42,13 @@ Author       NI
 About
 =====
 
-The **nimi-python** repository generates Python bindings (Application Programming Interface) for interacting with the Modular Instrument drivers. The
-following drivers are supported:
+This package, which is maintained in the `**nimi-python** repository <https://github.com/ni/nimi-python>`_, provides a Python API for the associated driver.
 
-* NI-DCPower (Python module: nidcpower)
-* NI-Digital Pattern Driver (Python module: nidigital)
-* NI-DMM (Python module: nidmm)
-* NI-FGEN (Python module: nifgen)
-* NI-ModInst (Python module: nimodinst)
-* NI-SCOPE (Python module: niscope)
-* NI Switch Executive (Python module: nise)
-* NI-SWITCH (Python module: niswitch)
-* NI-TClk (Python module: nitclk)
-
-It is implemented as a set of `Mako templates <http://makotemplates.org>`_ and per-driver metafiles that produce a Python module for each driver. The driver is
-called through its public C API using the `ctypes <https://docs.python.org/2/library/ctypes.html>`_ Python library.
-
-**nimi-python** supports all the Operating Systems supported by the underlying driver.
+Support Policy
+--------------
+This package supports all the Operating Systems supported by the underlying driver.
 
-**nimi-python** follows `Python Software Foundation <https://devguide.python.org/#status-of-python-branches>`_ support policy for different versions. At
+It follows `Python Software Foundation <https://devguide.python.org/#status-of-python-branches>`_ support policy for different versions. At
 this time this includes Python 3.7 and above using CPython.
```

Comment:
the second sentence is a maintenance burden, in my opinion. unless generated.

Feel free to remove.

Reply:
Handled in the PR where we dropped Python 3.7 support.